### DISCUSSION TITLE
有官方交流群么，感觉挺好用的。想交流下。

### DISCUSSION THREAD

**Discussion by bbhxwl** [youzan/vant#10407] [General]
有官方交流群么，感觉挺好用的。想交流下。

**Comment by chenjiahan** [comment]
![image](https://user-images.githubusercontent.com/7237365/158916872-1261b702-5d80-4684-baf4-7aaa5333613d.png)

**Reply by a798032825** [reply]
过期了，大佬能再发下嘛

**Reply by chenjiahan** [reply]
![image](https://user-images.githubusercontent.com/7237365/163988287-f9572264-8a71-422a-8e46-1a43965cfb3b.png)

**Comment by edwinhuish** [comment]
大佬，又过期了。。。能否再发一下？

**Comment by chenjiahan** [comment]
没必要加群了哈，群里都是灌水的居多，有相关问题需要讨论，直接在 GitHub 的 Discussions 里讨论就可以
